inst/WORDLIST

```diff
@@ -1,16 +1,19 @@
 CDISC
+cloneable
 Forkers
+formatters
+funder
 Getter
+getter
 Hoffmann
+iteratively
 JoinKeys
 Pre
-Reproducibility
-SCDA
-UI
-cloneable
-formatters
-funder
-iteratively
 pre
 repo
+Reproducibility
 reproducibility
+returens
+SCDA
+testthat
```

Comment:
why is this on the list?

Reply:
No idea. I've regenerated the WORDLIST on https://github.com/insightsengineering/teal.data/commit/1dc2db4d6c646e14ab6e82204108591bfb764de9 , some of the items were removed.

Thanks 🙇